torch/optim/__init__.py

```diff
@@ -13,6 +13,9 @@
 from .adamax import Adamax
 from .asgd import ASGD
 from .sgd import SGD
+from .sradam import SRAdam
+from .sradamw import SRAdamW
+from .srsgd import SRSGD
```

Comment:
It's unclear whether we want to expose stochastic rounding as separate optimizer classes or as constructor options to existing optimizers.  I don't think we need to make this choice now, though.  I think Facebook people should look at what you've got then we can decide together.